.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        javaversion: [ "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19" ]
+        javaversion: [ "11", "12", "13", "14", "15", "16", "17", "18", "19" ]
```

Comment:
Whoa... we need to build against Java 8 heh. We support it. Do we need to use an older version of Error Prone that supports it?

Reply:
That's what I am trying to figure out, Java VM was not able to create on Java 8 and it seems like it needs at least version 11 to create it: https://stackoverflow.com/questions/72045938/sonarqube-could-not-create-the-java-virtual-machine-unrecognized-option-add-e